src/Lift.ts

```diff
@@ -24,9 +23,11 @@ import { highlightDatamodel } from './cli/highlight/highlight'
 import { blue } from './cli/highlight/theme'
 import { DevComponentRenderer } from './ink/DevComponentRenderer'
 import { LiftEngine } from './LiftEngine'
+import { Studio } from './Studio'
 import { EngineResults, FileMap, LocalMigration, LocalMigrationWithDatabaseSteps, LockFile, Migration } from './types'
 import { drawBox } from './utils/drawBox'
 import { formatms } from './utils/formartms'
+import { getDatamodelPath } from './utils/getDatamodelPath'
```

Comment:
I created this utility by moving `Lift.getDatamodelPath` to a separate file, since I need to use the same functionality in Studio.ts